moderation/reports/README.md

```diff
@@ -38,12 +38,20 @@ Should you need further information because something is unclear, or want to rep
 Make for each custom command file a separate custom command, preferrably in the same category to keep them neat and organized. Please make sure to follow each step precisely and to use the correct trigger and trigger type, as well.
 
 #### Here's what you have to do:
-1. Disable the native report command, found here: `Control Panel > Tools & Utilities > Moderation`
+1. Disable the native report command, found here: `Control Panel > Tools & Utilities > Moderation`, like [this](https://discord.com/channels/766370592445956106/767771719720632350/795330585618022420)
```

Comment:
I'm unable to see where this message links to.

Reply:
Oh, this was supposed to link to an image, my bad :/